UnrealPlugin/ThirdPersonMPCloudDeploy.md

```diff
@@ -15,28 +15,81 @@ This guide explains how to deploy your game to PlayFab Multiplayer Server (MPS)
+* [OPTIONAL] This section can be done to refresh the files which Unreal builds into your project. This is useful if you've recently created or deleted files.
+	* If you have not recently built your project in "development editor" configuration, do so now
+	* Close Visual Studio
+	* Navigate to the .uproject file for your project
+	* Right click on the .uproject file in your file editor and select "Generate Visual Studio Project Files"	
+	* Re-open the {ProjectName}.sln file in Visual Studio
+* Choose "development editor" configuration
+* Run your project
+* Observe the Unreal Editor, which should have loaded your project
+* In the top left of the Unreal editor menu, click on File -> Package -> Target Configuration -> {ProjectName}Server
+* In the top left of the Unreal editor menu, click on File -> Package -> SOMETHING SOMETHING -> Shipping
+* File -> Package -> Windowsx64
+* Choose and remember an output location {ShippingServerFolder}
+* You can now use this packaged version of your game server to [test with LocalMultiplayerAgent](https://docs.microsoft.com/gaming/playfab/features/multiplayer/servers/locally-debugging-game-servers-and-integration-with-playfab)
+* Once verified with LocalMultiplayerAgent, consult the [creating a build guide](https://docs.microsoft.com/en-us/gaming/playfab/features/multiplayer/servers/deploying-playfab-multiplayer-server-builds)
+	* Testing on MPS at this point will require an intermediate tool from [PlayFab MpsAllocatorSample](https://github.com/PlayFab/MpsSamples/blob/master/MpsAllocatorSample/README.md)
+	* This tool bypasses some client work for now, and allows us to allocate a server without a direct request from the game client
+	* The proper way to do this on a client is a separate topic covered in our  [allocation guide.](https://docs.microsoft.com/en-us/gaming/playfab/features/multiplayer/servers/allocating-game-servers-and-configuring-vs-debugging-tools)
+* Zip your Shipping Server build from {ShippingServerFolder} above
+	* Your zip should contain everything in {ShippingServerFolder}, optionally at the root of the zip file
+	* [Optional] IE, the root of your zip should contain {ProjectName}Server.exe (It's suggested the exe should not be in a subfolder such as "WindowsServer". Rebuild your zip if desired)
+* Get familiar with the [build creation process guide](https://docs.microsoft.com/en-us/gaming/playfab/features/multiplayer/servers/deploying-playfab-multiplayer-server-builds)
+* Specific details for specific configurations are detailed below:
 
 ### Setting up a Windows Dedicated Server on PlayFab
 
-An important note about this is that you need to set the start command in process-based mode to \<root folder\>\\Binaries\\Win64\\\<project name\>Server.exe (or however the executable is called in the folder).
-
-In container-based mode use \<mount folder\>\\\<root folder\>\\Binaries\\Win64\\\<project name\>Server.exe (or however the executable is called in the folder).
```

Comment:
Did you find a fix for not needing to use the file in the binary folder?

Reply:
I have tested using the exe that is found in the root of the Staging builds, and it works, both in Process mode and in Container mode. The exe at the root invokes the exe in the Binaries/Win64 subfolder.